### DISCUSSION TITLE
How to make the install "real" and install a profile into the system?

### DISCUSSION THREAD

**Discussion by rschr** [eoyilmaz/displaycal-py3#223] [Q&A]
I've managed to build and run displaycal-py3 (yay! thanks a lot for reviving it) following the instructions, but when I install my display profile through the displaycal menu it doesn't survive reboot. Is there a trick I missed?

Also, every time I want to open the displaycal GUI again, I need to activate the virtual environment I created for the build process, where I would have hoped to have everything installed in my system including the system's menus. Is there a way to somehow move the application from "virtual" to "real" (sorry, developer here but complete Python noob)? 

The machine in question is running Linux Mint 21 Xfce based on Ubuntu 22.04 LTS.

**Comment by p5k369** [comment]
> it doesn't survive reboot. Is there a trick I missed?

Does it show in `xfce4-display-settings`? And have you tried setting the profile with `xfce4-display-settings` and does it still get reset after reboot?

> Is there a way to somehow move the application from "virtual" to "real" (sorry, developer here but complete Python noob)?

Technically you can install displaycal via superuser into your system. I would never do that myself (on a productive machine) and I do also not recommend that.
The virtual environment is not only for building the wheels, it enables you to run displaycal separated from the systems python environment.
Personally, I think it is better to install software into your system with the specific package manager a distribution provides because developers can make sure that a certain package or a related package of a certain package does not break other packages.

You do not need to activate the virtual environment explicitly. If you want to run a python script you can simply pass that to any interpreter you like (even one in a virtual environment). 
see https://realpython.com/python-virtual-environments-a-primer/#it-runs-from-anywhere-with-absolute-paths
